_data/changelogs/component-table.yml

```diff
@@ -4,6 +4,14 @@ type: component
 changelogURL:
 
 items:
+  - date: NNNN-NN-NN
+    summary: Fixed a bug that prevented `th` elements outside of `thead` from acquiring table heading styles.
+    summaryAdditional: Now, table headings will have consistent styles anywhere in the table element.
```

Comment:
**suggestion:** Thinking we should try to make the summary a bit more plain language. I also am not entirely sure this should be considered bug, so I removed that language. 

```suggestion
    summary: Updated table header styles to be consistent across all table elements.
    summaryAdditional: Now, all `thead th`, `tbody th`, and `tfoot th` cells will all have the same visual styles. Teams should confirm that their tables display as expected. 
```